apps/ui/src/views/Proposal/Votes.vue

```diff
@@ -159,31 +168,34 @@ watch([sortBy, choiceFilter], () => {
               : 'bg-skin-border opacity-40'
           "
         />
-        <div class="pl-4 py-3 w-[50%] lg:w-[40%] flex items-center gap-x-3 truncate">
+
+        <router-link
+          :to="{
+            name: 'user',
+            params: {
+              id: vote.voter.id
+            }
+          }"
+          class="leading-[22px] !ml-4 py-3 max-w-[218px] w-[218px] flex items-center space-x-3 truncate"
+        >
           <UiStamp :id="vote.voter.id" :size="32" />
-          <router-link
-            :to="{
-              name: 'user',
-              params: {
-                id: vote.voter.id
-              }
-            }"
-            class="overflow-hidden leading-[22px]"
-          >
+          <div class="flex flex-col truncate">
             <h4 class="truncate" v-text="vote.voter.name || shortenAddress(vote.voter.id)" />
             <div
               class="text-[17px] text-skin-text truncate"
               v-text="shortenAddress(vote.voter.id)"
             />
-          </router-link>
-        </div>
-        <div
-          class="hidden leading-[22px] w-[25%] lg:w-[20%] lg:flex flex-col justify-center truncate"
+          </div>
+        </router-link>
+        <a
+          class="grow w-[40%] flex flex-col items-start justify-center truncate leading-[22px] !cursor-default"
+          :class="{
+            'cursor-pointer': vote.reason
+          }"
```

Comment:
This doesn't work for me. I always see default cursor.

Maybe instead we just should make it `button` with `disabled` property?
This way we don't need handling `cursor` `tabindex` and `keypress`, it all works out of the box (cursor is pointer if not disabled, tabindex skips if it's disabled, pressing enter while focused triggers a click).

```vue
    <button
      :disabled="!vote.reason"
      class="grow w-[40%] flex flex-col items-start justify-center truncate leading-[22px]"
      @click="handleChoiceClick(vote)"
    >
```

Reply:
Fixed the issue, was just a CSS priority issue.

I would avoid transforming this to a BUTTON, due to the content (div, tooltip, etc ...)

Reply:
I feel like button is more well suited for this use-case, we have issue of abusing `a` links in the app I think still, but in this case we are using `a` and apply hacks to just make it work like a button.

Generally `a` links should be used for navigating to proper URL, if we add `onclick` event handler to `a` [it most likely should just be a `button`](https://developer.mozilla.org/en-US/docs/Learn/Accessibility/HTML#onclick_events).

There is nothing wrong with `button` containing dev, tooltip etc. any more than `a` link containing those elements as well. So I'd say we should use semantically correct element (a `button`) to just leverage built-in functionality instead of recreating it all from scratch.